src/periphery/SablierMerkleFactory.sol

```diff
@@ -36,17 +62,71 @@ contract SablierMerkleFactory is ISablierMerkleFactory {
         return totalPercentage == uUNIT;
     }
 
+    /// @inheritdoc ISablierMerkleFactory
+    function sablierFeeByUser(address campaignCreator)
+        external
+        view
+        override
+        returns (MerkleFactory.SablierFee memory)
+    {
+        return _sablierFeeByUser[campaignCreator];
+    }
+
+    /*//////////////////////////////////////////////////////////////////////////
+                         ADMIN-FACING NON-CONSTANT FUNCTIONS
+    //////////////////////////////////////////////////////////////////////////*/
+
+    /// @inheritdoc ISablierMerkleFactory
+    function resetSablierFeeByUser(address campaignCreator) external override onlyAdmin {
+        delete _sablierFeeByUser[campaignCreator];
+
+        // Log the reset.
+        emit ResetSablierFee({ admin: msg.sender, campaignCreator: campaignCreator });
+    }
+
+    /// @inheritdoc ISablierMerkleFactory
+    function setDefaultSablierFee(uint256 defaultFee) external override onlyAdmin {
+        // Effect: update the default Sablier fee.
+        defaultSablierFee = defaultFee;
+
+        emit SetDefaultSablierFee(msg.sender, defaultFee);
+    }
+
+    /// @inheritdoc ISablierMerkleFactory
+    function setSablierFeeByUser(address campaignCreator, uint256 fee) external override onlyAdmin {
```

Comment:
Given that this function will mostly be called by the admin either directly through etherscan or a contract call, wouldn't it be a good UX if we allow simpler datatypes to be the input parameters instead of a struct? I suppose Sablier UI wont integrate these functions.

Even though it means we need two functions `setSablierFeeByUser` and `resetSablierFeeByUser` whereas with your approach, we would need only `setSablierFeeByUser`.

My only argument in favour of keeping `setSablierFeeByUser` and `resetSablierFeeByUser` is that they are more explicit to what they do, leaves no room for errors (for example, an admin cannot accidentally pass `{fee: some number, enabled: false}`).

Reply:
> directly through etherscan or a contract call, wouldn't it be a good UX if we allow simpler datatypes to be the input parameters instead of a struct

correct, using a struct isn't the best experience, we could write it with direct params: `function setSablierFeeByUser(address campaignCreator, uint256 fee, bool enabled)` 

but as you said:
>> My only argument in favour of keeping setSablierFeeByUser and resetSablierFeeByUser is that they are more explicit to what they do, leaves no room for errors

which it makes sense and i agree with